src/containers/Installer.jsx

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+
+import InstallerView from '../components/InstallerView'
+import { installApp } from '../actions'
+
+const mapStateToProps = (state, ownProps) => ({
+  slug: state.slug,
```

Comment:
![](https://i.imgur.com/bcrfUCw.png)

— source [You might not need redux](https://medium.com/@dan_abramov/you-might-not-need-redux-be46360cf367)